File: logger.py
# -*- coding: utf-8 -*-

# helper functions

def wrap(text, *args):
    print("[%5s] " % text, end="")
    print(*args)


def info(*args):
    wrap("INFO", *args)


def error(*args):
    wrap("ERROR", *args)


def warn(*args):
    wrap("WARN", *args)


__all__ = ["info", "error", "warn"]

File: test-cs/tpepeip1/Logger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace tpepeip1
{
    public static class Logger
    {
        public static void Wrap(string text, params string[] args)
        {
            Console.Write("[" + text.PadLeft(5, ' ') + "] " + string.Join(" ", args));
        }

        public static void Info(params string[] args)
        {
            Wrap("INFO", args);
        }

        public static void Error(params string[] args)
        {
            Wrap("ERROR", args);
        }

        public static void Warn(params string[] args)
        {
            Wrap("WARN", args);
        }
    }
}
